docs/3rd-party/multibaas.md

```diff
@@ -0,0 +1,215 @@
+---
+id: multibaas
+title: MultiBaas by Curvegrid
+sidebar_label: MultiBaas
+---
+
+*By the end of the guide you should know how to interact with Ethereum smart contracts using MultiBaas tool provided by [Curvegrid](https://curvegrid.com/).*
+
+### Goals
+
+You should use this guide if you need to accomplish one of the following goals:
+- Mint (issue / generate) ERC20 tokens.
+- Burn (destroy) ERC20 tokens.
+- Deposit ERC20 tokens to the OMG Network.
+
+### Prerequisites
+
+1. Chrome or Brave browser.
+2. [MetaMask](https://MetaMask.io/download.html) wallet extension with $5-$10 worth of ETH.
+3. Basic understanding of Ethereum blockchain.
+
+### Overview
+
+Traditionally, minting and managing ERC20 tokens involves help from a blockchain engineer that understands smart contracts. However, the goal of the OMG Network is to eliminate this barrier and deliver a product that can be used by any enterprise client. 
+
+That's why we partnered with [Curvegrid](https://www.curvegrid.com/) to provide an easy solution for software companies that plan on working with the Ethereum blockchain and the OMG Network. Curvegrid platform offers a rich REST API for developers, as well as an intuitive administrative panel to manage various smart contracts for regular users.
+
+This guide demonstrates how users/admins can use the latter option to manage ERC20 tokens.
+
+### Issue ERC20 Tokens
+
+To issue ERC20 tokens using MultiBaas use the following steps:
+
+1. Create a [Curvegrid account](https://console.curvegrid.com/signup). 
+
+During the sign up you will need to choose the Ethereum network you want to work with:
+- Ropsten/Rinkeby - the Ethereum testnets. All of the transactions that happen on these networks are for test purposes only. The funds used on Ropsten or Rinkeby are not real.
+- Mainnet - the Ethereum mainnet. All of the transactions that happen on this network are for production purposes. The funds used on this network are real, so make sure you don't lose access to them.
+- Curvegrid - the network created by Curvegrid, won't be covered during this guide.
+
+2. Select `CONTRACTS -> Contracts` menu and choose `Mlti Token` contract:
```

Comment:
do we need to add more descriptions for this contract? maybe a brief descriptions of contract functionality and a link to the contract source if exists?

Reply:
Let's gather more internal feedback first and then add all the changes in bulk. I think there will be more clarifications for sure.